src/components/DepositWidget/useRowActions.tsx

```diff
@@ -65,8 +66,13 @@ export const useRowActions = (params: Params): Result => {
   }
 
   async function enableToken(tokenAddress: string): Promise<void> {
-    const { symbol } = getToken('address', tokenAddress, balances)
+    const { symbol } = getToken('address', tokenAddress, balances) as Required<TokenBalanceDetails>
```

Comment:
If I understood the docs (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html#improved-control-over-mapped-type-modifiers), `Required<TokenBalanceDetails>` removes the `?` properties.
Does that mean things like `symbol`, `name` and `decimals` will be stripped away?

Reply:
it's saying that the returned `TokenBalanceDetails` is the entire object with all `?` properties filled. Meaning the rest of the code relying on say, `sellTokenBalance.symbol` will never be `undefined`

Reply:
`Required<>` makes optional fields required, hence the name